sys/net/gnrc/network_layer/sixlowpan/frag/sfr/Kconfig

```diff
@@ -0,0 +1,91 @@
+
+if KCONFIG_USEMODULE_GNRC_SIXLOWPAN_FRAG_SFR
+
+config GNRC_SIXLOWPAN_SFR_MIN_FRAG_SIZE
+    int "Default minimum value for fragment size (MinFragmentSize)"
+    default 96
+
+config GNRC_SIXLOWPAN_SFR_MAX_FRAG_SIZE
+    int "Default maximum value for fragment size (MaxFragmentSize)"
+    default 112
+    help
+        It must be lower than the minimum MTU along the path. A large value
+        augments the chances of buffer bloat and transmission loss. The value
+        must be less than 512 if the unit is defined for the PHY layer is the
+        octet.
+
+config GNRC_SIXLOWPAN_SFR_OPT_FRAG_SIZE
+    int "Default value for fragment size that the sender should use to start with (OptFragmentSize)"
+    default 112
+
+config GNRC_SIXLOWPAN_SFR_USE_ECN
+    bool "Indicates whether the sender should react to ECN (UseECN)"
+    default false
+    help
+        When the sender reacts to ECN its window size will vary between @ref
+        CONFIG_GNRC_SIXLOWPAN_SFR_MIN_WIN_SIZE and @ref
+        CONFIG_GNRC_SIXLOWPAN_SFR_MAX_WIN_SIZE.
+
+if GNRC_SIXLOWPAN_SFR_USE_ECN
+comment "Warning: Reaction of sender to ECN is not implemented yet"
+endif
+
+config GNRC_SIXLOWPAN_SFR_MIN_WIN_SIZE
+    int "Default minimum value of window size that the sender can use (MinWindowSize)"
+    default 1
+    range 1 32
+
+config GNRC_SIXLOWPAN_SFR_MAX_WIN_SIZE
+    int "Default maximum value of window size that the sender can use (MaxWindowSize)"
+    default 16
+    range 1 32
+
+config GNRC_SIXLOWPAN_SFR_OPT_WIN_SIZE
+    int "Default value of window size that the sender should start with (OptWindowSize)"
+    default 16
+    range 1 32
+
+config GNRC_SIXLOWPAN_SFR_INTER_FRAME_GAP_US
+    int "Default minimum amount of time between transmissions in microseconds (InterFrameGap)"
+    default 100
+    help
+        All packets to a same destination, and in particular fragments, may be
+        subject to receive while transmitting and hidden terminal collisions
+        with the next or the previous transmission as the fragments progress
+        along a same path. The InterFrameGap protects the propagation of to one
+        transmission before the next one is triggered and creates a duty cycle
+        that controls the ratio of air and memory in intermediate nodes that a
+        particular datagram will use.
+
+config GNRC_SIXLOWPAN_SFR_MIN_ARQ_TIMEOUT_MS
+    int "Default minimum amount of time in milliseconds a node should wait for an RFRAG Acknowledgment before it takes a next action (MinARQTimeOut)"
+    default 350
+
+config GNRC_SIXLOWPAN_SFR_MAX_ARQ_TIMEOUT_MS
+    int "Default maximum amount of time in milliseconds a node should wait for an RFRAG Acknowledgment before it takes a next action (MaxARQTimeOut)"
+    default 700
+
+config GNRC_SIXLOWPAN_SFR_OPT_ARQ_TIMEOUT_MS
+    int "Default starting point of the value of the amount of time in milliseconds that a sender should wait for an RFRAG Acknowledgment before it takes a next action (OptARQTimeOut)"
```

Comment:
These prompts are really long and hard to read in the menuconfig interface. Would it be possible to split this into a prompt and help string?

Reply:
I was thinking about this too. Even as a `@brief` they are quite un-brief ;-). They were mostly taken verbatim from the then current version of the draft, but that is not a must. Will see how I can fix them.